question-servers/freeform.js

```diff
@@ -224,7 +224,8 @@ module.exports = {
             return courseExtensionsCache[course.id].data;
         }
 
-        let extensions = await module.exports.loadExtensionsAsync(path.join(course.path, 'elementExtensions'));
+        const coursePath = chunks.getRuntimeDirectoryForCourse(course);
+        let extensions = await module.exports.loadExtensionsAsync(path.join(coursePath, 'elementExtensions'));
```

Comment:
Came across this while verifying paths in this file - I *think* this is the right way to handle the intersection of extensions and chunks. Wanted to call this out to get a second opinion.

Reply:
Seems good to me. At some point it might be nice to do lazy-loading of extensions so we only take the hit on questions that use an element with extensions, but this is fine for now.